test/PushNotificationTests/BaseTestSuite.cpp

```diff
@@ -113,6 +114,27 @@ void BaseTestSuite::ChannelRequestUsingRemoteId()
     }
 }
 
+void BaseTestSuite::ChannelRequestCheckExpirationTime()
+{
+    if (PushNotificationManager::Default().IsSupported())
+    {
+        auto channelOperation{ PushNotificationManager::Default().CreateChannelAsync(c_azureRemoteId) };
+        VERIFY_SUCCEEDED(ChannelRequestHelper(channelOperation));
+
+        auto channel{ channelOperation.GetResults().Channel() };
+        auto expirationTime{ channel.ExpirationTime() };
+        auto expiryBound{ winrt::clock::now() + std::chrono::days(31) };
```

Comment:
Add a comment here that we want granularity within 31 days because we expect the expiry to be a maximum of 30 days from now

Reply:
If you want better granularity, you can use std::chrono:hours and have the expiration be std::chrono::hours(30*24 + 1). Then the granularity changes to a max of 1 hr delta.